ui/pages/bridge/prepare/prepare-bridge-page.test.tsx

```diff
@@ -272,3 +278,68 @@ describe('PrepareBridgePage', () => {
     expect(getByTestId('from-amount').closest('input')).toHaveValue('2.131');
   });
 });
+
+describe('useEnableMissingNetwork', () => {
+  const arrangeReactReduxMocks = () => {
+    jest
+      .spyOn(ReactReduxModule, 'useSelector')
+      .mockImplementation((selector) => selector({}));
+    jest.spyOn(ReactReduxModule, 'useDispatch').mockReturnValue(jest.fn());
+  };
```

Comment:
I tried using `renderHookWithProviderTyped`, but it kept crashing 😭 - so mocking redux 🤪